nvflare/apis/impl/controller.py

```diff
@@ -145,3 +145,16 @@ def cancel_task(
 
     def cancel_all_tasks(self, completion_status=TaskCompletionStatus.CANCELLED, fl_ctx: Optional[FLContext] = None):
         self.communicator.cancel_all_tasks(completion_status, fl_ctx)
+
+    def get_client_disconnect_time(self, client_name):
+        """Get the time that the client is deemed dead.
```

Comment:
Suggest change docstring to use same term (either "disconnected" or "dead") for consistency.

Just to confirm, it seems that they roughly mean the same thing, except disconnected occurs after a client has been reported dead for > dead_client_grace_period correct? Maybe using `_DeadClientStatus.disconnect_time` instead of `_DeadClientStatus.death_time` makes sense?

Reply:
Good catch. The reasons I wanted to change to "disconnected" are:
- There is really no way to know for sure that the client is dead. All we could tell is that the client is currently disconnected. Hence disconnected is more accurate.
- secondly, I don't like the term "death" in this public method that users may need to call. It is too dramatic and may even offend some people. I don't want to change other places that use the term "dead" since they are all internal code and won't be called by other developers.